aws-cdk/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "aws-cdk",
+  "version": "0.1.0",
+  "bin": {
+    "aws-cdk": "bin/aws-cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
```

Comment:
Is this cdk that apply the infrastructure as code in the rds-stack file?

Reply:
yes to deploy you 1) build, 2) maybe run cdk diff and then 3) cdk deploy to deploy the infrastructure